Examples/php/class/runme.php

```diff
@@ -14,7 +12,7 @@
 
 # ----- Access a static member -----
 
-print "\nA total of " . Shape::nshapes() . " shapes were created\n";
+print "\nA total of " . Shape::nshapes_get() . " shapes were created\n";
```

Comment:
In the PHP wrapper, the static nshapes() fn does both the work of calling nshapes_get() and nshapes_set(). So one less indirections here since we are nearly scraping what's generated at .php